tests/authentication/client/test_client_company.py

```diff
@@ -427,12 +427,12 @@ def test_can_get_all_reviews_made_by_a_user(self):
         self.assertEqual(res.status_code, status.HTTP_200_OK)
 
     def test_cannot_get_deleted_reviews(self):
-        """ user cannot get reviews that have been deleted """
+        """ user cannot get reviews that have been deleted"""
 
         client_id = self.company1.pk
         view = ClientReviewsView.as_view()
         url = reverse('auth:add-reviews', args=[client_id])
         response = self.factory.get(url)
         force_authenticate(response, user=self.user1)
         res = view(response, client_id=client_id)
-        self.assertEqual(res.status_code, status.HTTP_404_NOT_FOUND)
+        self.assertEqual(res.status_code, status.HTTP_200_OK)
```

Comment:
@dorothyas Why are you asserting `200` status code when testing for records that are not found?